multitenancy-spring-boot-autoconfigure/src/test/java/org/axonframework/extensions/multitenancy/autoconfig/MultiTenancyAutoConfigurationTest.java

```diff
@@ -121,6 +119,7 @@ void multiTenancyDisabled() {
                          assertThat(context).doesNotHaveBean(MultiTenantDeadLetterQueueFactory.class);
                          assertThat(context).doesNotHaveBean(MultiTenantEventScheduler.class);
                          assertThat(context).doesNotHaveBean(MultiTenantQueryUpdateEmitter.class);
+                         assertThat(context).doesNotHaveBean(TenantPersistentStreamMessageSourceFactory.class);
```

Comment:
Question: why not check if the `PersistentStreamMessageSourceFactory` is **not** present here too?

Reply:
Ah good to know. Thanks for the explanation!